ppdet/py_op/target.py

```diff
@@ -155,148 +167,152 @@ def generate_proposal_target(rpn_rois,
                              is_cascade_rcnn=False):
 
     rois = []
-    labels_int32 = []
-    bbox_targets = []
-    bbox_inside_weights = []
-    bbox_outside_weights = []
+    tgt_labels = []
+    tgt_deltas = []
+    rois_inside_weights = []
+    rois_outside_weights = []
     rois_nums = []
     batch_size = gt_boxes.shape[0]
```

Comment:
batch_size is not used here. Besides，mask target can be splited as well, and it needs to support batch_size larger than 1.